charts/govuk-apps-conf/templates/static-nginx-configmap.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: static-nginx-conf
+  labels:
+    {{- include "govuk-apps-conf.labels" . | nindent 4 }}
+    app.kubernetes.io/component: web
+data:
+  nginx.conf: |-
+    user nginx;
+
+    error_log  /var/log/nginx/error.log warn;
+    pid        /var/run/nginx.pid;
+
+    events {
+      worker_connections  1024;
+    }
+
+    http {
+      include       /etc/nginx/mime.types;
+      default_type  application/octet-stream;
+
+      server_tokens off;
+
+      sendfile        on;
+      keepalive_timeout  65;
+
+      upstream {{ .Release.Name }} {
+        server 127.0.0.1:{{ .Values.appPort }};
+      }
+
+      server {
+        listen {{ .Values.nginxPort }};
+
+        location / {
+          proxy_set_header   Host $http_host;
+          proxy_set_header   X-Real-IP $remote_addr;
+          proxy_pass         http://{{ .Release.Name }};
+          proxy_redirect     off;
+        }
+        location /assets {
+          proxy_set_header   Authorization "";
+          proxy_set_header   Connection "";
+          proxy_set_header   X-Real-IP $remote_addr;  # TODO: pass the actual end-client address
+          proxy_hide_header  x-amz-id-2;
+          proxy_hide_header  x-amz-meta-server-side-encryption;
+          proxy_hide_header  x-amz-request-id;
+          proxy_hide_header  x-amz-server-side-encryption;
+          proxy_hide_header  x-amz-version-id;
+          add_header         Cache-Control max-age=31536000;
+          proxy_intercept_errors on;
+          proxy_pass         https://govuk-app-assets-{{ .Values.govukEnvironment }}.s3.eu-west-1.amazonaws.com;
+        }
+      }
+
+      # Google site verification
+      location /googlec908b3bc32386239.html {
```

Comment:
This verification page is currently only served on assets.publishing.service.gov.uk (see [govuk-puppet](https://github.com/alphagov/govuk-puppet/blob/32c1bbb/modules/router/templates/assets_origin.conf.erb#L87)). Where does the `static` app come into this?

Reply:
At the moment the nginx config that does all this is on the cache nodes, and it sends the traffic by default to static: https://github.com/alphagov/govuk-puppet/blob/32c1bbb/modules/router/templates/assets_origin.conf.erb#L82-L85

I assumed this would be a good place to put it, since it's the default backend. From what I can see, k8s doesn't have an ingress for assets-origin yet, but I thought this would be where it would go to if it was to replicate existing configuration

Reply:
Oh, I see. In that case I think it might be in our interests to wait until we have assets-origin set up before adding these bits and pieces, then. Otherwise we're liable to add complexity without knowing whether it's needed.

Reply:
That makes sense to me. I'll update the trello card so nobody else tries to do the same thing

Reply:
Cool, thanks!